core/vm/logger_trace.go

```diff
@@ -15,92 +13,69 @@ var (
 		CALLCODE:     {traceToAddressCode, traceLastNAddressCode(1), traceContractAccount, traceLastNAddressAccount(1)}, // contract account is the caller, stack.nth_last(1) is the callee's address
 		DELEGATECALL: {traceToAddressCode, traceLastNAddressCode(1)},
 		STATICCALL:   {traceToAddressCode, traceLastNAddressCode(1), traceLastNAddressAccount(1)},
-		CREATE:       {}, // caller is already recorded in ExtraData.Caller, callee is recorded in CaptureEnter&CaptureExit
-		CREATE2:      {}, // caller is already recorded in ExtraData.Caller, callee is recorded in CaptureEnter&CaptureExit
-		SLOAD:        {}, // trace storage in `captureState` instead of here, to handle `l.cfg.DisableStorage` flag
-		SSTORE:       {}, // trace storage in `captureState` instead of here, to handle `l.cfg.DisableStorage` flag
 		SELFDESTRUCT: {traceContractAccount, traceLastNAddressAccount(0)},
 		SELFBALANCE:  {traceContractAccount},
 		BALANCE:      {traceLastNAddressAccount(0)},
 		EXTCODEHASH:  {traceLastNAddressAccount(0)},
-		CODESIZE:     {traceContractCode},
-		CODECOPY:     {traceContractCode},
-		EXTCODESIZE:  {traceLastNAddressCode(0)},
+		EXTCODESIZE:  {traceLastNAddressAccount(0)},
```

Comment:
why we don't need code here any more?

Reply:
account info struct has 6 fields,  {poseidon,keccak}_codehash, codesize, value, nonce, storage_root.  So to get 'codesize', the full bytecode is not needed in our l2geth